src/utils/link-handler.js

```diff
@@ -10,16 +10,18 @@
  * governing permissions and limitations under the License.
  */
 
-function link({ LINK_TITLES }) {
-  return function handler(h, node) {
-    const props = {};
-    if (node.url) {
-      props.href = node.url.replace(/\.md/, '.html');
-    }
-    if (LINK_TITLES) {
-      props.title = node.title || props.href;
+const fallback = require('mdast-util-to-hast/lib/handlers/link');
+const url = require('url');
+const clone = require('clone');
+
+
+function link() {
+  return function handler(h, node, children) {
+    const n = clone(node);
+    if (!url.parse(n.url).protocol) {
+      n.url = n.url.replace(/\.md/, '.html');
```

Comment:
Maybe just replace it in the path.

Reply:
You never know when someone registers the `.md` top level domain.

Reply:
good point

Reply:
Fixed in https://github.com/adobe/hypermedia-pipeline/pull/120/commits/5f705848fe86fd2f1726d7b98ed1ce07734a5b0d